lib/deploy/deploy-support.js

```diff
@@ -290,16 +290,27 @@ async function makeService({
     try {
       service = await fc.getService(serviceName);
     } catch (ex) {
-      
-      if (ex.code === 'AccessDenied' || ex.code === 'ERR_INVALID_CHAR' || ex.code === 'ENOTFOUND') {
-        console.error(red(`The accountId entered is incorrect. We speculate that you used the subuser uid, login name, login name prefix, display name or thers\n`+
-        `please enter the correct accountId, which is the account ID under your primary account. You can reset your account ID by executing fun config\n`+
-        `For the method of taking the primary account ID, please refer to https://account.console.aliyun.com/#/secure`));
+
+      if (ex.code === 'AccessDenied') {
+
+        if (ex.message.indexOf('the caller is not authorized to perform') !== -1) {
+
+          console.error(red(`\nMaybe you can grant AliyunRAMFullAccess policy to the subuser or use the primary account. You can refer to Chinese doc https://github.com/aliyun/fun/blob/master/docs/usage/faq-zh.md#nopermissionerror-you-are-not-authorized-to-do-this-action-resource-acsramxxxxxxxxxxrole-action-ramgetrole or English doc https://github.com/aliyun/fun/blob/master/docs/usage/faq.md#nopermissionerror-you-are-not-authorized-to-do-this-action-resource-acsramxxxxxxxxxxrole-action-ramgetrole for help.\n\nIf you don’t want use the AliyunRAMFullAccess policy or primary account, you can also specify the Role property for Service. You can refer to Chinease doc https://github.com/aliyun/fun/blob/master/docs/specs/2018-04-03-zh-cn.md#aliyunserverlessservice or Enaglish doc https://github.com/aliyun/fun/blob/master/docs/specs/2018-04-03.md#aliyunserverlessservice for help.\n`));
+
+        } else if (ex.message.indexOf('FC service is not enabled for current user') !== -1) {
+
+          console.error(red(`FC service is not enabled for current user. Please enable FC service before using fun.\nYou can enable FC service on this page https://www.aliyun.com/product/fc`));
+
+        } else {
+          console.error(red(`The accountId entered is incorrect. We speculate that you used the subuser uid, login name, login name prefix, display name or thers\n`+
```

Comment:
The accountId entered is incorrect. You must use the primary account id. You can get primary account ID on this page please refer to https://account.console.aliyun.com/#/secure .